src/github/nag.rs

```diff
@@ -224,6 +224,7 @@ fn evaluate_nags() -> DashResult<()> {
         };
 
         let num_active_reviews = reviews.iter().filter(|&&(_, ref r)| !r.reviewed).count();
```

Comment:
Maybe this should be renamed to `num_outstanding_reviews` or `num_pending_reviews` to contrast it with complete reviews?